src/bin/pg_autoctl/primary_standby.c

```diff
@@ -733,6 +733,11 @@ standby_init_replication_source(LocalPostgresServer *postgres,
 
 	if (primaryNode != NULL)
 	{
+		upstream->primaryNode.nodeId = primaryNode->nodeId;
+
+		strlcpy(upstream->primaryNode.name,
+				primaryNode->name, _POSIX_HOST_NAME_MAX);
```

Comment:
Does `name` have a maximum of `_POSIX_HOST_NAME_MAX`? I would expect `MAX_BUFSIZE`.

Reply:
Indeed, our node name has the same length as a hostname:

``` c
	char name[_POSIX_HOST_NAME_MAX];
	char hostname[_POSIX_HOST_NAME_MAX];
```